android/src/main/kotlin/com/mapbox/maps/mapbox_maps/MapboxMapController.kt

```diff
@@ -79,6 +85,9 @@ class MapboxMapController(
     FLTSettings.LocationComponentSettingsInterface.setup(messenger, null)
     FLTSettings.LogoSettingsInterface.setup(messenger, null)
     FLTSettings.GesturesSettingsInterface.setup(messenger, null)
+    FLTSettings.CompassSettingsInterface.setup(messenger, null)
```

Comment:
nit:
maybe add to some list and call `setup` in loop?

Reply:
Not possible since `setup` is not an interface method but static one implemented for every generated controller by Pigeon